src/components/PasswordField/Indicator/passwordFieldIndicator.scss

```diff
@@ -9,9 +9,11 @@ $password-field-indicator-height: 6px;
   position: absolute;
   right: $form-field-white-horizontal-padding;
   width: 60px;
-  top:
-    $form-field-padding-top +
-    (($form-field-white-font-size - $password-field-indicator-height) / 2);
+  top: 30px;
```

Comment:
we have password fields on blue and gray screens (inputs with different height), so `top` couldn't be fixed value

Reply:
They are the same and equal to 65px